homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -650,6 +656,18 @@ async def wake_word_detection(
                 self.pipeline.wake_word_id,
             )
 
+            if result is not None:
+                last_wake_up = self.hass.data.get(DATA_LAST_WAKE_UP)
```

Comment:
Can we move this code down to the existing if-check on result.

Reply:
Moved